### ISSUE TITLE
\keyval_parse:NNn isn't marked as rEXP correctly

### ISSUE THREAD

**Issue by Skillmon** [latex3/latex3#833]
Currently `\keyval_parse:NNn` is marked as `EXP` in the documentation, since it leaves every list element in `\exp_not:n` in the input stream this isn't the case and it should be marked as `rEXP`.

In https://github.com/latex3/latex3/pull/832 this is already fixed.

**Comment by josephwright** [comment]
I don't think `\exp_not:n` would be an issue, but having a loop that doesn't 'push' all the results to appearing at one point is. (For example, `\text_uppercase:n` is `f`-type expandable but uses a single `\exp_not:n`.)

**Comment by Skillmon** [comment]
Yes, badly formulated. That's what I meant. It doesn't push the output to the end, but just leaves `\exp_not:n` for each element of the key=value list in the input stream.

**Comment by josephwright** [comment]
The code was merged a while ago, so I'll close.
